### ISSUE TITLE
[Android] Google suspended my app because of a trademark violation against WhatsApp(?)

### ISSUE THREAD

**Issue by MarvinJWendt** [WhatsApp/stickers#150]
Hello, 
### **First of all this isn´t related to the coding part of this repo**.
(I still think that this issue has to be here because I couldn´t get any response via email and this could happen to others aswell. Maybe there should be a clarification in the readme on how to avoid such a thing from happening.)

Now to the little story:
I made my own sticker app using this template without modifying any design or layout.
Now google sent me a "trademark violation notice" via email. They stated that I am using WhatsApp´s logo, and that´s violating the trademark. They removed the app, and if they remove my other sticker apps aswell, my Developer Account will get suspended.

I really don´t get where the issue is. The logos are in the button "Add to WhatsApp" but I didn´t put them there, the template has them in already. I haven´t used WhatsApp´s logo anywhere else.

Now I am confused on what to do, because using this offical template which is even suggested by the WhatsApp FAQ shouldn´t get developers banned.

In my app I also stated that I am in no connection to the WhatsApp company and that it is an independent WhatsApp Sticker Addon.

Now my question is: Is there any way to prove that the app is a fork from an official template that we may use? Also I would really be happy if my app gets unsuspended.
( applicationId: com.MarvinJWendt.MegaMemeStickerPack )

Additional information given to me by Google:
`trademark_relationship: Note: AppDetex is authorized by WhatsApp to 
facilitate the submission of and correspondence regarding complaints.`

`trademark_explain: The app uses the trademarks, banners, badges and logos 
of WhatsApp without authorization.`

PS: Or could there be something added on "what not to do" when publishing sticker apps, to avoid getting take downs?

Grettings
Marvin Wendt

**Comment by simonzhexu** [comment]
Thanks for letting us know. And sorry for the frustrating experience. We will get back to you on this.

**Comment by MarvinJWendt** [comment]
Thank you @simonzhexu for your quick reply. If you need any further information let me know :)

**Comment by GlowMLG** [comment]
You seem to have been affected by a hoax. Check issue #190

**Comment by sdergachev** [comment]
We have the same situation. How to get WhatsApp approval? Already wrote a few letters, no answer.

**Comment by AnasCode** [comment]
> Check issue #190

@GlowMLG Can we use Whatsapp in the name ?

**Comment by Zandor300** [comment]
@AnasCode You are allowed to use WhatsApp in the name of your app, as long as you follow the guidelines: https://whatsappbrand.com

Example name that should work: [My Sticker App Name] for WhatsApp

If you have a name like that. The issue lies elsewhere and the WhatsApp team is working on this with Google.

**Comment by AnasCode** [comment]
would it be possible to use it in the future ?

**Comment by simonzhexu** [comment]
> Hey, sorry for the late reply, I was on vacation.
> As @Zandor300 mentioned the name "WhatsApp" may not be used in the Apps name.
> 
> I for myself think that this is something which should be worked on. Those sticker apps are literally WhatsApp addons. I think it is false advertising to **not** write the name into the title as this suggests that the stickers can be used in every app (as jpg or something).
> This is just my opinion and maybe we will see a change in the future.
> I have taken all my sticker apps offline for now, to avoid getting my personal Google account terminated.
> 
> Thank you for your assistance in this matter,
> Marvin Wendt

This is a valid concern. But since we have to live under play store rules. We don't suggest you to use WhatsApp app name.

**Comment by MoshDev** [comment]
@MarvinJWendt did you get any reply from them? where you able to republish it later?

**Comment by MarvinJWendt** [comment]
@MoshDev, actually I still have all my sticker Apps offline. As I see it right now, you can publish the app, if you don't put WhatsApp in the title :)

**Comment by MoshDev** [comment]
@MarvinJWendt  i mean the app that got suspended, were you able to republish it? did you contact AppDetex and got response from them? sorry, my app got suspended few hours ago and im searching for a solution like crazy

**Comment by MarvinJWendt** [comment]
@MoshDev I indeed wrote with AppDetex, but they did not allow me to republish the app. Except if I "correct" the mistake.
Pretty sure they just sent me a template answer.

**Comment by Lurzapps** [comment]
got suspended as well. This is my third app being banned from the PlayStore now. The first one was a mistake I made when I was 14, the other one was a CoronaTracker. I have been banned probably as well because I used the brand's name in my title. What a shit. I have to upload it completely again, they did not even send me any kind of information why it had been banned. This is absolutely not great for small developers, they just get crushed by such things. Well, I guess I'll upload it again without the brand's name in the title.
